shell/probe/raindrop.ps1

```diff
@@ -1,11 +1,20 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
-function Execute { 
+function Execute {
     Param([String]$File)
 
     try {
-        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru).ExitCode
+        $stdoutTempFile = New-Item -path "$dir\stdout.log" -Force
+        $stderrTempFile = New-Item -path "$dir\stderr.log" -Force
+        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru -RedirectStandardOutput $stdoutTempFile -RedirectStandardError $stderrTempFile).ExitCode
         $Code = if (Test-Path $File) {$R} Else {127}
+        $stdout = Get-Content -Path $stdoutTempFile
+        $stdout = if ($stdout) { [string]::Join("; ", $stdout) } else { "" }
+        $stderr = Get-Content -Path $stderrTempFile
+        $stderr = if ($stderr) { [string]::Join("; ", $stderr) } else { "" }
+        if ($stdout -or $stderr) {
+            Write-Host "${stdout}; ${stderr}"
+        }
         return $Code
     } catch [System.UnauthorizedAccessException] {
         return 126
```

Comment:
will things still get written if you end up in here? (/ did they ever?)

Reply:
oh no they will not... I'm going to stick the log stuff in a finally block instead